packages/@aws-cdk/aws-cloudfront/lib/distribution.ts

```diff
@@ -3,6 +3,7 @@ import { Construct, IResource, Lazy, Resource, Stack, Token, Duration } from '@a
 import { CfnDistribution } from './cloudfront.generated';
 import { Origin } from './origin';
 import { CacheBehavior } from './private/cache-behavior';
+import { SSLMethod, SecurityPolicyProtocol } from './web_distribution';
```

Comment:
One more thing -- I've been incrementally moving these sorts of common enums from './web_distribution' to this file, as the (very long-term) plan is to deprecate web_distribution in favor of this construct. Bonus points if you move these two over as part of this change. :D